### PR TITLE
Add hobbit generator.

### PR DESCRIPTION
**Pull request by jeroendevrind**
Hi @vadymmarkov ,

I try to contribute with the very important generator class - Hobbit ;-)

Please take a look :)

### CODE REVIEW

**Comment by dunyakirkali** [comment]
@jeroendevrind Could you also update the `README.md` with some usage instructions?

File: Resources/Locales/en.json [hunk 1]
```diff
@@ -9855,7 +9855,27 @@
           "#{Name.name}",
           "#{Company.name}"
         ]
-      }
+      },
+      "hobbit": [
+        {
+          "character":["Bilbo Baggins", "Bungo Baggins", "Belladonna Took", "Bullroarer Took", "Gandalf The Grey", "Radagast", "Dain", "Thorin Oakenshield", "Fili", "Kili", "Balin", "Dwalin", "Oin", "Gloin", "Dori", "Nori", "Ori", "Bifur", "Bofur", "Bombur", "Elrond", "Galion", "Bard the Bowman", "Beorn", "Tom", "Bert", "William (Bill Huggins)", "Gollum", "The Necromancer", "Smaug", "Carc", "Roac", "The Lord of the Eagles", "The Great Goblin", "Bolg", "Golfimbul"],
```
Comment (dunyakirkali) [inline Resources/Locales/en.json]: @jeroendevrind Might be a bit more legible to have each array item on a new line

File: Resources/Locales/en.json [hunk 2]
```diff
@@ -9855,7 +9855,27 @@
...
+          "thorins_company":["Thorin Oakenshield", "Fili", "Kili", "Balin", "Dwalin", "Oin", "Gloin", "Dori", "Nori", "Ori", "Bifur", "Bofur", "Bombur", "Gandalf", "Bilbo Baggins"],
```
Comment (dunyakirkali) [inline Resources/Locales/en.json]: @jeroendevrind Might be a bit more legible to have each array item on a new line

File: Resources/Locales/en.json [hunk 3]
```diff
@@ -9855,7 +9855,27 @@
...
+            "Where there's life there's hope.",
+            "So comes snow after fire, and even dragons have their endings.",
+            "Where did you go to, if I may ask?' said Thorin to Gandalf as they rode along.  To look ahead,' said he.  And what brought you back in the nick of time?' Looking behind,' said he.",
+            "'You have nice manners for a thief and a liar,' said the dragon.",
+            "May the hair on your toes never fall out!",
+            "The road goes ever on and on",
+          "Never laugh at live dragons, Bilbo you fool!"],
+          "location": ["Bree", "The Shire", "Rivendell", "The Misty Mountains", "Beorn's Hall", "Mirkwood", "Esgaroth", "Erebor", "Bag-End", "Under-Hill", "Mount Gram", "Green Fields", "Last Desert", "Lonely Mountain", "Withered Heath", "Country Round", "Long Lake", "River Running", "Mines of Moria", "Green Dragon Inn", "Bywater", "The Great Mill", "Wilderland", "Gondolin", "Land Beyond", "Goblin Gate", "Carrock", "High Pass", "Great River", "Grey Mountains", "Land of the Necromancer", "Long Marshes", "Forest River", "Lake Town", "Dorwinion", "Ravenhill", "Iron Hills", "Mount Gundabad"]
```
Comment (dunyakirkali) [inline Resources/Locales/en.json]: @jeroendevrind Might be a bit more legible to have each array item on a new line

File: Tests/Fakery/Generators/HobbitSpec.swift [hunk 4]
```diff
@@ -0,0 +1,52 @@
+//
```
Comment (dunyakirkali) [inline Tests/Fakery/Generators/HobbitSpec.swift]: @jeroendevrind Could u please remove the copyright?

File: Sources/Fakery/Generators/Hobbit.swift [hunk 5]
```diff
@@ -0,0 +1,27 @@
+//
```
Comment (dunyakirkali) [inline Sources/Fakery/Generators/Hobbit.swift]: @jeroendevrind Could u please remove the copyright?

**Comment by vadymmarkov** [comment]
@jeroendevrind Very important indeed 😅 Could you also add this generator to https://github.com/vadymmarkov/Fakery/blob/master/Sources/Fakery/Faker.swift to make it accessible.

**Comment by vadymmarkov** [comment]
@jeroendevrind Do you mind fixing merge conflicts?

**Comment by jeroendevrind** [comment]
> @jeroendevrind Do you mind fixing merge conflicts?

I did. Can u merge?

**Comment by vadymmarkov** [comment]
Thanks @jeroendevrind!

**Pull request merged**